corehq/apps/ota/tests/digest_restore.py

```diff
@@ -5,6 +5,13 @@
 from django_digest.test import Client
 
 
+def setupModule():
+    from unittest import SkipTest
+    raise SkipTest("this is broken, hasn't been run by django test runner. Error: "
```

Comment:
Looks like we do https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/ota/tests/__init__.py#L1 Should I remove it?